src/main/java/nextstep/subway/favorite/application/FavoriteService.java

```diff
@@ -0,0 +1,47 @@
+package nextstep.subway.favorite.application;
+
+import static java.util.stream.Collectors.toList;
+
+import java.util.List;
+import java.util.Optional;
+import nextstep.subway.favorite.domain.Favorite;
+import nextstep.subway.favorite.domain.FavoriteRepository;
+import nextstep.subway.favorite.dto.FavoriteCreateRequest;
+import nextstep.subway.favorite.dto.FavoriteResponse;
+import nextstep.subway.station.application.StationService;
+import nextstep.subway.station.domain.Station;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+@Service
+@Transactional
+public class FavoriteService {
+    private final FavoriteRepository favoriteRepository;
+    private final StationService stationService;
+
+    public FavoriteService(FavoriteRepository favoriteRepository,
+                           StationService stationService) {
+        this.favoriteRepository = favoriteRepository;
+        this.stationService = stationService;
+    }
+
+    @Transactional(readOnly = true)
+    public List<FavoriteResponse> findFavorites(Long memberId) {
+        List<Favorite> favoriteList = favoriteRepository.findByMemberId(memberId);
+        return favoriteList.stream().map(FavoriteResponse::of).collect(toList());
```

Comment:
```suggestion
        return favoriteList.stream()
        .map(FavoriteResponse::of)
        .collect(toList());
```

스트림은 행위 단위로 내려쓰는 것이 가독성과 디버깅등 유지보수에 좋다고 생각하는데
어떻게 생각하시나요?

Reply:
동의합니다 ㅎㅎ